app/oess_pull_azure_interfaces.pl

```diff
@@ -88,6 +113,70 @@ sub reconcile_oess_endpoints {
     }
 }
 
+sub find_matching_azure_connection{
+    my $azure_connection_info = shift;
+    my $cloud_interconnect_id = shift;
+    my $azure_peering_subnet;
+    my $peering = $azure_connection_info->{properties}->{peerings};
+    foreach my $ip (@$peering){
+        if($cloud_interconnect_id =~ m/-SEC-/){
+            $azure_peering_subnet = $ip->{properties}->{secondaryPeerAddresssubnet};
+        }elsif($cloud_interconnect_id =~ m/-PRI-/){
+            $azure_peering_subnet = $ip->{properties}->{primaryPeerAddresssubnet};
+        }else{
+            $azure_peering_subnet = "";
```

Comment:
Returning `undef` might make more sense here?

Reply:
no returning undef instead of ""